src/github/context.ts

```diff
@@ -2,8 +2,21 @@ import {context} from '@actions/github'
 import {ContextBlock} from '@slack/web-api'
 import {Duration, formatDuration} from 'date-fns' // eslint-disable-line import/named
 import {link} from '../slack/mrkdwn'
-import {Link} from '../slack/types'
-import {isPullRequestEvent} from './webhook'
+import {Image, Link} from '../slack/types'
+import {
+  isPullRequestEvent,
+  isSupportedEvent,
+  SupportedEventName,
+  UnsupportedEventError
+} from './webhook'
+
+export const EVENT_NAME_IMAGE_MAP: Record<SupportedEventName, string> = {
+  pull_request:
+    'https://user-images.githubusercontent.com/847532/193414326-5aaf5449-0c81-4a66-9b19-4e5e6baeee9e.png',
+  push: 'https://user-images.githubusercontent.com/847532/193413878-d5fcd559-401d-4954-a44c-36de5d6a7adf.png',
+  schedule:
+    'https://user-images.githubusercontent.com/847532/193414289-3b185a3b-aee8-40f9-99fe-0615d255c8dd.png'
+} as const
```

Comment:
<img width="419" alt="Screen Shot 2022-10-02 at 9 54 45 AM" src="https://user-images.githubusercontent.com/847532/193480113-9f8a37ae-b6bb-4c33-a07e-3b32609990ea.png">

<img width="429" alt="Screen Shot 2022-10-02 at 9 54 56 AM" src="https://user-images.githubusercontent.com/847532/193480118-c667ebe3-7582-447e-a1a7-c991417b2229.png">

I added a semi-transparent stroke such that these are somewhat visible in dark mode, but they're optimized for light mode.

Reply:
wow, I love this image idea!

Reply:
Where are you getting these url from?

Reply:
I uploaded them into some GitHub markdown comment and hope they'll just live there indefinitely. Do you have concerns or suggestions on a better place to host them?

Reply:
O I see. That seems like the best strategy to keep this working indefinitely